### DISCUSSION TITLE
Version 3.0 as Docker

### DISCUSSION THREAD

**Discussion by twixwix** [netbox-community/netbox-docker#545] [Ideas]
I saw that version 3 is launched in beta state. Is there any plan to create a docker out of it?

**Comment by ryanmerolle** [comment]
Currently we have @tobiasge https://quay.io/repository/tgenannt/netbox?tab=tags

It is built from feature branch instead of the tag [v3.0-beta1](https://github.com/netbox-community/netbox/tree/v3.0-beta1)

It should work just fine, but I have not tested recently.

**Reply by twixwix** [reply]
thanks a lot for the quick reply.
